new/LIT7055Qene.xml

```diff
@@ -0,0 +1,75 @@
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7055Qene" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ልጓመ፡ ወቅንዋተ፡ በአማን፡ ተወክፈ፡ ፋሲለደስ፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Lǝgʷāma wa-qǝnwāta ba-ʾamān tawakkǝfo Fāsiladas (Qǝne of wāzemā type)</title>
+                <title xml:lang="en" corresp="#t1">Fāsiladas in true faith receiving the halter and the spurs ...</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="BNFabb145"/>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Qǝne of the wāzemā-type. Type and author are not indicated in the manuscript.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="Poetry"/>
+                    <term key="Qene"/>
+                    <term key="Wazema"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-08-31">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="BNFabb145"/> on f. 55rc. Division into verses is indicated with
+                    <foreign xml:lang="gez">።</foreign>. The end of the poem is indicated with <foreign xml:lang="gez">። ።</foreign>.</note>
+                <ab>
+                    <l n="1">ልጓመ፡ ወቅንዋተ፡ በአማን፡ ተወክፈ፡ <persName ref="PRS4015Fasilada">ፋሲለደስ፡</persName> መለኮትሰ፡ ልጓመ፡ ዘይቤ፡ ውስተ፡ ገጽየ፡ ፈነወ፡ ኢዮብ፡ በእንተ፡ ደዊሁ፡ ኅዳጠ፡ ሐሰወ።</l>
```

Comment:
ደዌየ

Reply:
I see ደዊሁ፡/ደዌሁ፡ in the first verse. You mean, I should change to ደዌሁ፡ ?

<img width="359" alt="Screenshot 2024_Qene7055 (II)" src="https://github.com/user-attachments/assets/d4357eea-63b9-4f9c-9c23-d27c6782a614">

Reply:
Yes, sorry! If you compare it to the "wi" two lines down, I think it's clear that it is "we" (which is also what we would expect, I suppose)